src/shared_modules/content_manager/src/actionOrchestrator.hpp

```diff
@@ -36,7 +36,25 @@ class ActionOrchestrator final
     enum UpdateType
     {
         CONTENT,
-        OFFSET
+        OFFSET,
+        FILE_HASH
+    };
+
+    /**
+     * @brief Struct containing the necessary members to execute the orchestrations.
+     *
+     */
+    struct UpdateData
+    {
+        UpdateType type = UpdateType::CONTENT; ///< Orchestration update type.
+        int offset = -1;                       ///< Offset value used in the update.
+        std::string fileHash;                  ///< Hash value used in the update.
```

Comment:
I think the documentation could be improved with the valid values and what each one means (-1, 0, >0, etc)

Reply:
Yes. Done.